hlsclt/build_commands/build_commands.py

```diff
@@ -43,11 +43,19 @@ def do_start_build_stuff(ctx):
 def do_default_build(ctx):
     config = ctx.obj.config
     file = ctx.obj.file
+    solution_num = ctx.obj.solution_num
     file.write("csim_design -clean" + (" -compiler clang" if config.get("compiler","") == "clang" else "") + " \n")
     file.write("csynth_design" + "\n")
     file.write("cosim_design -O -rtl " + config["language"] + "\n")
     file.write("export_design -format ip_catalog" + "\n")
     file.write("export_design -format sysgen" + "\n")
+    # Copy the src/ files as well as the config file to keep track of the changes over solutions
+    import shutil
```

Comment:
Can we move this to the top please to keep all the imports together?

Reply:
Addressed at https://github.com/benjmarshall/hlsclt/pull/12/commits/4f279abe517401a65221606f37b487c1848e26a5.